plugins/catalog-backend/src/database/tables.ts

```diff
@@ -66,6 +66,7 @@ export type DbFinalEntitiesRow = {
   hash: string;
   stitch_ticket: string;
   final_entity?: string;
+  last_updated_at: string | Date;
```

Comment:
Oh one more thing. This needs to be `last_updated_at?` since it's optional

Reply:
Good catch! Changed